main.py

```diff
@@ -231,6 +230,38 @@ def kcidb_send_notification(data, context):
     message = SPOOL_CLIENT.pick(notification_id)
     if not message:
         return
+    # log message before sending email
+    LOGGER.info("SENDING %s", notification_id)
+    # send message via email
+    send_message(message)
+    # Acknowledge notification as sent
+    SPOOL_CLIENT.ack(notification_id)
+
+
+def kcidb_pick_notifications():
+    """
+    Pick abandoned notifications and resend them
```

Comment:
We're not "resending" them, because we haven't sent them yet. We're just sending them. Please fix. Also please put a full stop at the end of the sentence (we probably need to add pydocstyle to CI).